apps/solana_app/solana_helpers.h

```diff
@@ -0,0 +1,64 @@
+/**
+ * @file    solana_helpers.h
+ * @author  Cypherock X1 Team
+ * @brief   Utilities api definitions for Solana chains
+ * @copyright Copyright (c) 2023 HODL TECH PTE LTD
+ * <br/> You may obtain a copy of license at <a href="https://mitcc.org/"
+ * target=_blank>https://mitcc.org/</a>
+ */
+#ifndef SOLANA_HELPERS_H
+#define SOLANA_HELPERS_H
+
+/*****************************************************************************
+ * INCLUDES
+ *****************************************************************************/
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#include "coin_utils.h"
+
+/*****************************************************************************
+ * MACROS AND DEFINES
+ *****************************************************************************/
+
+// Derivation path reference :
+// https://docs.solana.com/wallet-guide/paper-wallet#hierarchical-derivation
+// Taking 3 levels of depth similar to ledger
+#define SOLANA_PUB_KEY_SIZE 33
```

Comment:
please recheck if this public key size of `33` or `32`. should be 32 IMO (refer: https://github.com/Cypherock/x1_wallet_firmware/blob/refactor/core-framework/apps/near_app/near_pub_key.c#L80).

Reply:
Yes, it should be 32. I'll make the changes